app/uk/gov/hmrc/customs/financials/views/components/eori_banner.scala.html

```diff
@@ -5,12 +5,16 @@
 
 @this()
 
-@(eori: Option[String] = None)(implicit messages: Messages)
+@(eori: Option[String] = None, companyName: Option[String] = None)(implicit messages: Messages)
 
 @eori.map { eoriNumber =>
     <div class="govuk-phase-banner">
         <p class="govuk-phase-banner__content">
+            @if(companyName eq None) {
             <span id="eori" class="govuk-phase-banner__text">@messages("cf.header.eori") @eoriNumber</span>
+            }else{
+            <span id="eoriWithCompanyName" class="govuk-phase-banner__text">@companyName @messages("cf.header.eori.dash") @eoriNumber</span>
```

Comment:
Not sure this will render correctly as you need to get the company name out of the Option.